lib/rucio/transfertool/fts3.py

```diff
@@ -685,7 +685,7 @@ def _is_recoverable_fts_overwrite_error(cls, request: dict[str, Any], reason: st
         dst_type = file_metadata.get('dst_type', None)
         METRICS.counter('overwrite.check.{rsetype}.{rse}').labels(rse=file_metadata["dst_rse"], rsetype=dst_type).inc()
 
-        if 'Destination file exists and overwrite is not enabled' in (reason or ''):
+        if 'Destination file exists and overwrite is not enabled' in (reason or '') or 'Destination file exists and is on tape' in (reason or ''):
```

Comment:
I'm a bit confused by the pre-existing check:
```[string] in (reason or '')```

`reason` is a `str`, and `bool(reason)` would only return `False` if it's empty, in which case it'd  just be: 
```[string] in ('' or '')```

If we expect `request` to optionally be `None`, then we should change the type annotation from `reason: str` to `reason: Optional[str]`, and we can rewrite the check like this:

```suggestion
        if reason and any(s in reason for s in ('Destination file exists and overwrite is not enabled', 'Destination file exists and is on tape')):
```

If we expect `request` to always be a valid `str`, then we can simplify to this:

```suggestion
        if any(s in reason for s in ('Destination file exists and overwrite is not enabled', 'Destination file exists and is on tape')):
```